Vwitter/Models/Vent.h

```diff
@@ -0,0 +1,22 @@
+//
+//  Vent.h
+//  Vwitter
+//
+//  Created by Christina Li on 7/12/22.
+//
+
+#import <Parse/Parse.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface Vent : PFObject<PFSubclassing>
+
+@property (nonatomic, strong) PFUser *author;
+
+@property (nonatomic, strong) NSString *ventContent;
```

Comment:
note: One cool stretch feature would be to support rich text (ie. ability to bold, italicize, etc.), I am sure there must be a good cocoapod that does this, the "rich" text view would probably expose an HTML or markdown-encoded string so that this model would not even need to change